api/interfaces/http/handlers/auth_handler.go

```
@@ -0,0 +1,423 @@
+	return resp, nil
+}
+
+// Logout clears authentication cookies and blacklists tokens
+func (h *AuthHandler) Logout(ctx context.Context, input *LogoutInput) (*LogoutOutput, error) {
+	// Add operation context for error tracking
+	ctx = errors.WithOperation(ctx, "logout")
+
+	// Try to blacklist current tokens before clearing cookies
+	// Blacklist access token if present
+	if input.AccessToken != "" {
+		if err := h.authMiddleware.BlacklistToken(ctx, input.AccessToken, "logout"); err != nil {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)